packages/api/src/users/users.repository.ts

```diff
@@ -0,0 +1,35 @@
+import { Injectable } from '@nestjs/common';
+
+import type { User } from '@coderscamp/shared/models/user';
+
+import { PrismaService } from '../prisma/prisma.service';
+
+@Injectable()
+export class UsersRepository {
```

Comment:
Shouldn't it be rather a service than a repository?

Reply:
Looks like repository for me. Just data retrieving / updating.